components/studio/bin/hab-studio.sh

```diff
@@ -895,18 +778,18 @@ Please retry this command as a super user or use a privilege-granting facility s
 # ```
 exit_with() {
   if [ "${HAB_NOCOLORING:-}" = "true" ]; then
-    printf -- "ERROR: $1\n"
+    echo "ERROR: $1"
   else
     case "${TERM:-}" in
       *term | xterm-* | rxvt | screen | screen-*)
-        printf -- "\033[1;31mERROR: \033[1;37m$1\033[0m\n"
+        echo "\033[1;31mERROR: \033[1;37m$1\033[0m"
         ;;
       *)
-        printf -- "ERROR: $1\n"
+        echo "ERROR: $1"
         ;;
     esac
   fi
-  exit $2
+  exit "$2"
```

Comment:
While shellcheck leads you to do this, in this case `$2` would be an integer which doesn't necessarily need to be quoted.

Reply:
Yeah, it's just easier to add the quotes than the `disable`, I figure. I mean, there's no harm, right?

Plus, if someone misused `exit_with` and did something like:
```bash
status="*" # <== assume a logic error leads to something like this
exit_with "bad news" "$status"
```
I'd say we don't want to expand that. This error:
```
ERROR: bad newslogout
bash: exit: *: numeric argument required
```
Is more understandable to me than:
```
ERROR: bad newslogout
bash: exit: builder: numeric argument required
```